src/music/bemidi.cpp

```diff
@@ -19,45 +19,68 @@
 #include "../safeguards.h"
 
 /** The file we're playing. */
-static BMidiSynthFile midiSynthFile;
+static BMidiSynthFile *midiSynthFile;
 
 /** Factory for BeOS' midi player. */
 static FMusicDriver_BeMidi iFMusicDriver_BeMidi;
 
+static double currentVolume;
```

Comment:
Coding style wise variables in the global scope need to start with an underscore, and they should not use camel casing. I know the midiSynthFile violates both. See https://wiki.openttd.org/en/Development/Coding%20style for more information.

Since you were wondering about the mismatch in drivers between static and instance variables, would you be up to make midiSynthFile, justStarted and currentVolume variables of MusicDriver_BeMidi? While doing that, please rename them to respectively midi_synth_file, just_started and current_volume and add some doxygen. Please be aware that any access to those class instance variable should use `this->`, e.g. `this->just_started`.

Normally I'm tempted to just do the changes myself, but I do not have an environment to test the changes and then I'd run the risk you having to fix my attempt to fix the coding style.